doc/api/esm.md

```diff
@@ -1130,18 +1130,45 @@ node --experimental-top-level-await b.mjs # works
 <!-- type=misc -->
 
 To customize the default module resolution, loader hooks can optionally be
-provided via a `--experimental-loader ./loader-name.mjs` argument to Node.js.
-
+provided via `--experimental-loader ./loader-name.mjs` arguments to Node.js
 When hooks are used they only apply to ES module loading and not to any
 CommonJS modules loaded.
 
+Loaders are modules which export functions and data which can be used by
+Node.js to modify the behavior of loading ES Modules. Loaders may be "chained"
+together by passing multiple `--experimental-loader` arguments to Node.js. A
+loader module will be imported once as a singleton.
+
+A hook has access to an additional `parentLoader` argument, which contains
+functions which call the hooks of the parent loader. When calling a parent hook,
+`parentLoader` **does not** need to be passed
+
+### <code>parentLoader</code>
+
+* {Object}
+  * `resolve` {Function}
+  * `getFormat` {Function}
+  * `getSource` {Function}
+  * `transformSource` {Function}
+
+When calling a parent hook, `parentLoader` **does not** need to be passed.
```

Comment:
One things that may be interesting to decide (not necessarily in this PR): Should these be methods or functions/bound methods ("is destructuring of this object safe")?

Reply:
Good point, it would be nice to maintain the `this` binding for plugin calls if they want to manage state that way.